targets/targets.json

```diff
@@ -285,7 +295,10 @@
         "release_versions": ["2", "5"],
         "features": ["LWIP"],
         "device_name": "LPC1768",
-        "bootloader_supported": true
+        "bootloader_supported": true,
+        "overrides": {
+            "network-default-interface-type": "ETHERNET"
```

Comment:
This should be CELLULAR by default.

Reply:
Why?
C027 does have ethernet, and ethernet does not require any configuration. Therefore it should be the default.

For cellular to work, you need to provide configuration.

Reply:
Cellular can work without configuration - we have a default APN database.